cmd/bnsd/x/account/codec.proto

```diff
@@ -128,11 +142,13 @@ message RegisterDomainMsg {
   bytes admin = 3 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
   // Has Superuser is a feature switch flag.
   bool has_superuser = 4;
-  // Third party token is an arbitrary data that can be provided by a middleman
+  // Broker is an arbitrary data that can be provided by a middleman
   // (i.e. retailer) that although does not participate directly in the
   // transaction, helped a user to register a domain. Storing a third party
-  // token helps identify contribution of such party later.
-  bytes third_party_token = 5;
+  // token helps identify contribution of such party later. Must be an email or
+  // a weave address that starts with a format or hex.
+  // example bech32:tiov16hzpmhecd65u993lasmexrdlkvhcxtlnf7f4ws
```

Comment:
Broker is a weave address (bech32 or hex) that can be provided by a middleman that helped facilitate the registration transaction.  For example, an IOV token holder that registers a domain in exchange for fiat from a client is a broker.  Storing the broker helps identify the contribution of such a party, which allows for automated commission distribution through an IOV reward initiative, for example.  Must be a weave address that starts with a format or hex, for example: bech32:tiov16hzpmhecd65u993lasmexrdlkvhcxtlnf7f4ws.